### ISSUE TITLE
Instance of 'minified:e' gives an error when I implement the Theme.dart into a project.

### ISSUE THREAD

**Issue by pelinyigit** [rxlabz/panache#11]
I was trying to understand the implementation of Panache. Since it is a pretty new tool, I am not quite sure if I am doing the implementation right, but I created a new theme on Panache Web and I changed the Primary swatch just to test if it would work with my project but when I download the Theme.dart class and implement it to my Flutter project's lib folder, I get an error for a specific part of the code. 

This is the code piece from Theme.dart:

![code](https://user-images.githubusercontent.com/37120828/64335098-c4091980-cfe2-11e9-8a15-1f078a4eaf98.JPG)
 

It gives an error for every  **Instance of 'minified:e'** 

This is the error I get: 

![error](https://user-images.githubusercontent.com/37120828/64335360-514c6e00-cfe3-11e9-8d83-359c2516dc2f.JPG)

Did I do something wrong while I was implementing the code in my project? or is it something else?

**Comment by gazialankus** [comment]
I get the same thing while using Panache Web. It seems Panache is outputting errors instead of the color values there. I just change the main color and the generated code has these `minified e` stuff, which look like errors. @rxlabz were you actually able to use the generated code in Flutter projects?

**Comment by rxlabz** [comment]
Sorry it was a bug. It's fixed now !

**Comment by pelinyigit** [comment]
Hi @rxlabz , thanks for the fix. It worked for the part that I posted a screenshot of, but actually, the code has lots of errors like this : 

![1](https://user-images.githubusercontent.com/37120828/64413319-f88bdc80-d099-11e9-84ea-8e56f27b183c.JPG)

![2](https://user-images.githubusercontent.com/37120828/64413328-fd509080-d099-11e9-87b1-24249510dc75.JPG)

![3](https://user-images.githubusercontent.com/37120828/64413334-004b8100-d09a-11e9-8170-ec0cb7f02ba2.JPG)

So, I just saw a Github repo for Panache Web, sorry for opening the issue in here. I realized main.dart.js has been edited. Is that the one that fixed the first error?
